pkgs/top-level/all-packages.nix

```diff
@@ -35669,6 +35669,8 @@ with pkgs;
 
   check-uptime = callPackage ../servers/monitoring/plugins/uptime.nix { };
 
+  check-zfs = callPackage ../servers/monitoring/plugins/zfs.nix { };
```

Comment:
If this is a Nagios plugin, it should be added to `servers/monitoring/nagios/plugins`.